docs/detections/rules-ui-create.asciidoc

```diff
@@ -136,6 +150,55 @@ per rule execution:
 ** _Field_: Leave blank
 ** _Threshold_: `10`
 
+* To create a new EQL rule, select _Event Correlation_ and then:
```

Comment:
👍 to @peluja1012 's [comment](https://github.com/elastic/security-docs/pull/272/files#r501724674) above: the inconsistency leads to weirdness like this.